### ISSUE TITLE
Use an actual enum field in databases that support it

### ISSUE THREAD

**Issue by ulope** [hzdg/django-enumfields#35]
It would be very nice if actual 'enum' type would be used for databases that support them.

**Comment by akx** [comment]
Hmm. That should be configurable, though; the CharField or IntField enums we have now are nice since changing the enum will not require a database migration whereas "physical" enum fields naturally would.

Feel free to write a PR though? :)

**Comment by ryanhiebert** [comment]
Yeah, the database migration issue is what ends up being really annoying, in my experience. For example, you might be using an enum from a 3rd party package, so it's annoying when that causes a database migration in your app, because it's not easy to tell that it is going to happen until you start seeing warnings when you open up django shells. It makes it easy for it to go out of sync.
